src/config/scrivitoContentBrowser.js

```diff
@@ -1,107 +1,152 @@
 import * as Scrivito from "scrivito";
 
 Scrivito.configureContentBrowser({
-  filters: {
+  filters: filterContext => {
+    const validObjClasses = filterContext._validObjClasses;
+    if (validObjClasses) {
+      switch (validObjClasses.length) {
+        case 0:
+          return invalidObjClassFilter();
```

Comment:
One could provide a obj class with a non existing restriction:

```js
poster: ["reference", { only: ["DoesNotExist"] }],
```

In this case `_validObjClasses` will be an empty array. If we use `objClassesFilter([])` in this case, the Content Browser does not render completely:

<img width="1323" alt="empty_valid_objs_classes" src="https://user-images.githubusercontent.com/86275/51983566-49e74400-2499-11e9-827f-a54bafa38d4b.png">

It also console logs the following error:

```
scrivito_ui.js:168 Error: Missing value (query term: {"field"=>"_obj_class", "operator"=>"equals", "value"=>[]})
```

We could also show `defaultFilter()` if `_validObjClasses.length === 0`.

Reply:
If this was a common case, raising an error would be the best option, because the original intention can not be fulfilled.

But I don't expect this to happen often, so I can also live with reverting to the default filter.